streampipes-client-python/streampipes/functions/utils/data_stream_generator.py

```diff
@@ -117,3 +136,35 @@ def create_data_stream(
 
     data_stream.element_id = sanitized_stream_id
     return data_stream
+
+# TODO Use an more general approach to create a data stream
+@deprecated('deprecated; please use "compose_data_stream" instead.')
```

Comment:
Could you please adapt the tests that use `create_data_stream` as well?
In addition, it would be great to have some dedicated tests for the new `compose_data_stream` function, if you are open to it